buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -80,21 +77,17 @@ class DistributedPerformanceTest extends PerformanceTest {
     @PathSensitive(PathSensitivity.RELATIVE)
     File scenarioList
 
-    @OutputFile
-    @PathSensitive(PathSensitivity.RELATIVE)
-    File scenarioReport
-
     @OutputDirectory
     @PathSensitive(PathSensitivity.RELATIVE)
     File reportDir
 
     private RESTClient client
 
-    private List<String> scheduledBuilds = Lists.newArrayList()
+    private List<String> scheduledBuildIds = Lists.newArrayList()
 
-    private List<Object> finishedBuilds = Lists.newArrayList()
+    private Map<String, Object> finishedBuilds = [:]
 
-    private Map<String, List<JUnitTestSuite>> testResultFilesForBuild = [:]
+    private Map<String, JUnitTestSuite> testResultForBuild = [:]
```

Comment:
Seems like each individual test only generates one xml, so no need to use `List` here.